widget/entry_validation.go

```diff
@@ -27,13 +36,13 @@ func (e *Entry) SetOnValidationChanged(callback func(error)) {
 	e.onValidationChanged = callback
 }
 
-// SetValidationError manually updates the validation status until the next input change
-func (e *Entry) SetValidationError(err error) {
+// SetValidationError manually updates the validation status until the next input change. Returns if the validation error is really set.
+func (e *Entry) SetValidationError(err error) bool {
```

Comment:
You are breaking the public facing API here. Please revert this change.

Reply:
OK